numba/core/config.py

```diff
@@ -306,6 +306,11 @@ def avx_default():
         DISABLE_JIT = _readenv("NUMBA_DISABLE_JIT", int, 0)
 
         # choose parallel backend to use
```

Comment:
This comment does not match this variable, it belongs to `THREADING_LAYER`.

Reply:
But shouldn't both `THREADING_LAYER_PRIORITY` and `THREADING_LAYER` are part of the "choose parallel backend to use" process?

Reply:
I think that the `THREADING_LAYER`, if prescribed, will act as the choice for the parallel backend in use, whereas the `THREADING_LAYER_PRIORITY` sets preference for use of threading layers in the case that no explicit choice is prescribed?